samples/utils/CommandLineUtils.cpp

```diff
@@ -539,7 +541,7 @@ namespace Utils
         returnData.input_clientId =
             cmdUtils.GetCommandOrDefault(m_cmd_client_id, Aws::Crt::String("test-") + Aws::Crt::UUID().ToString());
         returnData.input_thingName = cmdUtils.GetCommandRequired(m_cmd_thing_name);
-        returnData.input_jobId = cmdUtils.GetCommandRequired(m_cmd_job_id);
+        returnData.input_jobId = cmdUtils.GetCommandOrDefault(m_cmd_job_id, "1");
```

Comment:
Debatable: Set to empty string by default. And in the samples request for a job description only if jobId is set.